dev/src/Helpers.ts

```diff
@@ -329,35 +328,35 @@ export enum SubmissionSort {
 }
 
 /** Sorting */
-const enrollmentCompare = (a: Enrollment, b: Enrollment, sortBy: EnrollmentSort, descending: boolean): number => {
+const enrollmentCompare = (a: Enrollment.AsObject, b: Enrollment.AsObject, sortBy: EnrollmentSort, descending: boolean): number => {
     const m = descending ? -1 : 1
```

Comment:
@JosteinLindhom Would `sortOrder` be a good replacement for `m`? (same issue as in `state.tsx`)